README.adoc

```diff
@@ -138,9 +135,7 @@ For this application, a JSON structure is to be returned. The desired `Content-T
 response is `application/json` with `MediaType.APPLICATION_JSON` instead of the `String` content type. Using a constant such as `MediaType.APPLICATION_JSON` is better because if there's a spelling error, a compile failure occurs.
 
 JAX-RS supports a number of ways to marshal JSON. The JAX-RS 2.1 specification mandates JSON-Binding
-(JSON-B) and JAX-B. 
-
-The method body returns the result of `System.getProperties()` that is of type `java.util.Properties`. Since the method 
+(JSON-B) and JAX-B. The method body returns the result of `System.getProperties()` that is of type `java.util.Properties`. Since the method 
```

Comment:
The spec does mandate JAXB, but JAXB is intended for Java-to-XML (and vice versa) transformations, not Java-to-JSON.  I would suggest removing the reference to it here.

Reply:
Addressed.